### PR TITLE
Save active cart when starting to edit a quote

### PR DESCRIPTION
**Pull request by steinsebastian**

### CODE REVIEW

File: feature-libs/quote/core/facade/quote.service.spec.ts [hunk 1]
```diff
@@ -160,6 +165,7 @@ describe('QuoteService', () => {
   let cartUtilsService: CartUtilsService;
   let globalMessageService: GlobalMessageService;
   let quoteStorefrontUtilsService: QuoteStorefrontUtilsService;
+  let savedCartService: SavedCartFacade;
```
Comment (Larisa-Staroverova) [inline feature-libs/quote/core/facade/quote.service.spec.ts]: Please, rename to `savedcartFacade`

File: feature-libs/quote/core/facade/quote.service.ts [hunk 2]
```diff
@@ -75,6 +76,7 @@ export class QuoteService implements QuoteFacade {
   protected cartUtilsService = inject(CartUtilsService);
   protected globalMessageService = inject(GlobalMessageService);
   protected quoteStorefrontUtilsService = inject(QuoteStorefrontUtilsService);
+  protected savedCartService = inject(SavedCartFacade);
```
Comment (Larisa-Staroverova) [inline feature-libs/quote/core/facade/quote.service.ts]: Please, rename to `savedCartFacade`

**Review by Larisa-Staroverova — CHANGES_REQUESTED** [review]

File: feature-libs/quote/core/facade/quote.service.spec.ts (hunk 1 above)
Comment (steinsebastian) [reply feature-libs/quote/core/facade/quote.service.spec.ts]: Done

File: feature-libs/quote/core/facade/quote.service.ts (hunk 2 above)
Comment (steinsebastian) [reply feature-libs/quote/core/facade/quote.service.ts]: Done

**Review by Larisa-Staroverova — APPROVED** [review]

**Pull request merged**